modules/core/src/scenegraph/base-model.js

```diff
@@ -329,6 +329,40 @@ export default class BaseModel extends ScenegraphNode {
       vs = vs || MODULAR_SHADERS.vs;
       fs = fs || MODULAR_SHADERS.fs;
 
+      vs = `
```

Comment:
Can this go into mesh-model for now?

Reply:
Need to decide if we will have a dedicated shader for glTF.